vernac/comInductive.ml

```diff
@@ -255,7 +258,7 @@ let solve_constraints_system levels level_bounds =
   done;
   v
 
-let inductive_levels env evd poly arities inds =
+let inductive_levels env evd arities inds =
   let destarities = List.map (fun x -> x, Reduction.dest_arity env x) arities in
   let levels = List.map (fun (x,(ctx,a)) ->
     if Sorts.is_prop a || Sorts.is_sprop a then None
```

Comment:
We should not do the "is_flexible_sort" stuff that handles "natural" props in the template_poly case, as that turns inductives in Prop when they were in Type@{i} for any i I think. That is why 3125 fails I think.

Reply:
I'm a bit lost. Wasn't the `is_flexible_sort` stuff made precisely for template polymorphic inductives, that didn't really need to be?

Reply:
Nope, I think the is_flexible_sort stuff is rather for inferring the sort of "natural" propositions in Prop, but template polymorphic ones should stay in Type@{i} I believe. At least that's how I understand 3125.